lib/mfg.c

```diff
@@ -392,6 +399,26 @@ static int security_config_get_gen5(struct switchtec_dev *dev,
 	memcpy(state->public_key, reply.public_key,
 	       state->public_key_num * SWITCHTEC_KMSK_LEN);
 
+	state->attestation_supported = true;
+	state->attn_state.attestation_mode =
+		(reply.cfg >> SWITCHTEC_ATTEST_BITSHIFT) &
+		SWITCHTEC_ATTEST_BITMASK;
+	state->attn_state.uds_selfgen =
+		(reply.cfg >> SWITCHTEC_UDS_SELFGEN_BITSHIFT) &
+		SWITCHTEC_UDS_SELFGEN_BITMASK;
```

Comment:
Should probably be more specific here. Nothing links the values in `enum switchtec_attestation_mode` with the hardware values.  And they probably shouldn't be linked.